opcua.cpp

```diff
@@ -1040,7 +1040,7 @@ void OPCUA::setTraceFile(const std::string &traceFile)
 {
 	if (traceFile == "True" || traceFile == "true" || traceFile == "TRUE")
 	{
-		string traceFilePath = getDataDir() + string("/logs/");
+		string traceFilePath = getDataDir() + string("/logs/debug-trace/");
```

Comment:
This change is insufficient, as the `debug-trace` directory needs to be created by the plugin itself if it doesn't already exist. 

Based on the logs, when the debug trace option is enabled, no readings are ingested.

```
Dec  3 12:00:28 AJ FogLAMP s2[190291]: FATAL: Unable to initialise S2OPC CommonHelper library: 1
Dec  3 12:00:28 AJ FogLAMP s2[190291]: ERROR: plugin_start exception: Unable to initialise S2OPC CommonHelper library
```